packages/dev/core/src/assetContainer.ts

```diff
@@ -434,7 +443,7 @@ export class AssetContainer extends AbstractScene {
             } else {
                 // Mesh or TransformNode
                 let canInstance = true;
-                if (node.getClassName() === "TransformNode" || (node as Mesh).skeleton || (node as Mesh).getTotalVertices() === 0) {
+                if ((node.getClassName() !== "Mesh" && node.getClassName() !== "InstancedMesh") || (node as Mesh).skeleton !== null || (node as Mesh).getTotalVertices() === 0) {

```

Comment:
We don't have to test for the "InstancedMesh" because it is already taken care in the first branch of the `if`.

Also, there are other types of mesh that can be instantiated, like `LinesMesh` (which extends `Mesh`), so we can't test for `(node.getClassName() !== "Mesh"`.

I think the safest is:

```javascript
if (node.getClassName() === "TransformNode" || node.getClassName() === "Node" || (node as Mesh).skeleton || (node as Mesh).getTotalVertices() === 0) {
```